mulog-core/src/com/brunobonacci/mulog/jvm_metrics.clj

```diff
@@ -0,0 +1,66 @@
+(ns ^{:author "Pablo Reszczynski (@Pablo Reszczynski)"
+      :doc "Module for sampling some JVM metrics"}
+  com.brunobonacci.mulog.jvm-metrics
+  (:require [clojure.spec.alpha :as s])
+  (:import (java.lang.management MemoryMXBean
+                                 MemoryPoolMXBean)))
```

Comment:
please follow the https://github.com/bbatsov/clojure-style-guide
use square brackets for `import`
 
``` Clojure
(:import [java.lang.management MemoryMXBean
                               MemoryPoolMXBean])
```

Reply:
done